### ISSUE TITLE
[Version 3.0] Can not enable or change user passwords

### ISSUE THREAD

**Issue by sephra-dev** [nilsteampassnet/TeamPass#2847]
### Steps to reproduce
1. Login as admin
2. Create a new user
3. Try to login as the new user

### Expected behaviour
User should be created with an initial password

### Actual behaviour
No password can be set for new created users, password change is also not possible with following message:

`Teampass has been upgraded to a new version which includes a new encryption protocol. It expects from you to re-encrypt your personal items. For this, you need to provide your current Personal Salt Key (PSK). Once odne, please click on button Launch and wait until the process reached the final step. This could take several minutes depending on the number of objects. `

### Server configuration
**Operating system**: debian 10

**Web server:** apache 2

**Database:** mysql

**PHP version:** 7.3

**Teampass version:** 3.0.0.5

**Teampass configuration file:**

$SETTINGS = array (
    'max_latest_items' => '10',
    'enable_favourites' => '1',
    'show_last_items' => '1',
    'enable_pf_feature' => '1',
    'log_connections' => '0',
    'log_accessed' => '0',
    'time_format' => 'H:i:s',
    'date_format' => 'd/m/Y',
    'duplicate_folder' => '0',
    'item_duplicate_in_same_folder' => '0',
    'duplicate_item' => '0',
    'number_of_used_pw' => '3',
    'manager_edit' => '1',
    'cpassman_dir' => '/var/www/TeamPass-3.0.0.5',
    'cpassman_url' => 'https://url',
    'favicon' => 'https://url',
    'path_to_upload_folder' => '/var/www/TeamPass-3.0.0.5/upload',
    'path_to_files_folder' => '/var/www/TeamPass-3.0.0.5/files',
    'url_to_files_folder' => 'https://url/files',
    'activate_expiration' => '0',
    'pw_life_duration' => '0',
    'maintenance_mode' => '0',
    'enable_sts' => '0',
    'encryptClientServer' => '1',
    'cpassman_version' => '3.0.0.0',
    'ldap_mode' => '0',
    'ldap_type' => 'windows',
    'ldap_suffix' => 'DC=xxx;DC=xxx',
    'ldap_domain_dn' => 'CN=users,DC=xxx,DC=xxx,DC=xxx',
    'ldap_domain_controler' => 'xxx.xxx.xxx.xxx',
    'ldap_user_attribute' => '0',
    'ldap_ssl' => '0',
    'ldap_tls' => '0',
    'ldap_elusers' => '0',
    'ldap_search_base' => 'de',
    'ldap_port' => '389',
    'richtext' => '0',
    'allow_print' => '0',
    'roles_allowed_to_print' => '0',
    'show_description' => '1',
    'anyone_can_modify' => '0',
    'anyone_can_modify_bydefault' => '0',
    'nb_bad_authentication' => '0',
    'utf8_enabled' => '1',
    'restricted_to' => '0',
    'restricted_to_roles' => '0',
    'enable_send_email_on_user_login' => '0',
    'enable_user_can_create_folders' => '0',
    'insert_manual_entry_item_history' => '0',
    'enable_kb' => '1',
    'enable_email_notification_on_item_shown' => '0',
    'enable_email_notification_on_user_pw_change' => '0',
    'custom_logo' => '',
    'custom_login_text' => '',
    'default_language' => 'english',
    'send_stats' => '0',
    'send_statistics_items' => 'stat_country;stat_users;stat_items;stat_items_shared;stat_folders;stat_folders_shared;stat_admins;stat_managers;stat_ro;stat_mysqlversion;stat_phpversion;stat_teampassversion;stat_languages;stat_kb;stat_suggestion;stat_customfields;stat_api;stat_2fa;stat_agses;stat_duo;stat_ldap;stat_syslog;stat_stricthttps;stat_fav;stat_pf;',
    'send_stats_time' => '1603896187',
    'get_tp_info' => '1',
    'send_mail_on_user_login' => '0',
    'nb_items_by_query' => 'auto',
    'enable_delete_after_consultation' => '0',
    'enable_personal_saltkey_cookie' => '0',
    'personal_saltkey_cookie_duration' => '31',
    'email_smtp_server' => '',
    'email_smtp_auth' => '',
    'email_auth_username' => '',
    'email_auth_pwd' => '',
    'email_port' => '',
    'email_security' => '',
    'email_server_url' => '',
    'email_from' => '',
    'email_from_name' => '',
    'pwd_maximum_length' => '40',
    'google_authentication' => '0',
    'delay_item_edition' => '0',
    'allow_import' => '1',
    'proxy_ip' => '',
    'proxy_port' => '',
    'upload_maxfilesize' => '10mb',
    'upload_docext' => 'doc,docx,dotx,xls,xlsx,xltx,rtf,csv,txt,pdf,ppt,pptx,pot,dotx,xltx',
    'upload_imagesext' => 'jpg,jpeg,gif,png',
    'upload_pkgext' => '7z,rar,tar,zip',
    'upload_otherext' => 'sql,xml',
    'upload_imageresize_options' => '1',
    'upload_imageresize_width' => '800',
    'upload_imageresize_height' => '600',
    'upload_imageresize_quality' => '90',
    'use_md5_password_as_salt' => '0',
    'ga_website_name' => 'TeamPass for ChangeMe',
    'api' => '0',
    'subfolder_rights_as_parent' => '0',
    'show_only_accessible_folders' => '0',
    'enable_suggestion' => '1',
    'otv_expiration_period' => '7',
    'default_session_expiration_time' => '60',
    'duo' => '0',
    'enable_server_password_change' => '0',
    'ldap_object_class' => '0',
    'bck_script_path' => '/var/www/TeamPass-3.0.0.5/backups',
    'bck_script_filename' => 'bck_teampass',
    'syslog_enable' => '0',
    'syslog_host' => 'localhost',
    'syslog_port' => '514',
    'manager_move_item' => '0',
    'create_item_without_password' => '0',
    'otv_is_enabled' => '1',
    'agses_authentication_enabled' => '0',
    'item_extra_fields' => '0',
    'saltkey_ante_2127' => 'none',
    'migration_to_2127' => 'done',
    'files_with_defuse' => 'done',
    'timezone' => 'UTC',
    'enable_attachment_encryption' => '1',
    'personal_saltkey_security_level' => '50',
    'ldap_new_user_is_administrated_by' => '0',
    'disable_show_forgot_pwd_link' => '1',
    'offline_key_level' => '0',
    'enable_http_request_login' => '0',
    'ldap_and_local_authentication' => '1',
    'secure_display_image' => '1',
    'upload_zero_byte_file' => '0',
    'upload_all_extensions_file' => '0',
    'bck_script_passkey' => 'xxx',
    'admin_2fa_required' => '1',
    'password_overview_delay' => '4',
    'copy_to_clipboard_small_icons' => '1',
    'duo_akey' => '',
    'duo_ikey' => '',
    'duo_skey' => '',
    'duo_host' => '',
    'teampass_version' => '',
    'roles_allowed_to_print_select' => '',
    'clipboard_life_duration' => '30',
    'ldap-test-config-username' => 'xxx',
    'ldap-test-config-pwd' => 'xxx',
    'ldap_new_user_role' => '2',
);

**Updated from an older Teampass or fresh install:**
PLEASE attach to this issue the file `/includes/config/tp.config.php`.

Fresh Install

### Client configuration
**Browser:**
Firefox

**Operating system:**
Win10

#### Web server error log
```
Insert your webserver log here
```

#### Log from the web-browser developer console (CTRL + SHIFT + i)
```
Insert the log here and especially the answer of the query that failed.
```

**Comment by sephra-dev** [comment]
1. SMTP has to be correctly set.
2. E-Mail for user has to be correctly set.
3. Creating a new user will then send a password to the user e-mail
4. User can change password after login

=> Issue can be closed.

**Comment by V1kt0r87** [comment]
@sephra-dev 

hi! 

SMTP settings - OK
Create user = receive message with passsword - OK
User login - OK

but where can i **change my password**?? i can't find this option

**Comment by sephra-dev** [comment]
@V1kt0r87 

Hey,

you can change your password after login in the upper right corner in your profile settings.

**Comment by V1kt0r87** [comment]
![image](https://user-images.githubusercontent.com/75116524/103758972-49f00700-5013-11eb-955e-87d573f1e10e.png)

**Comment by V1kt0r87** [comment]
@sephra-dev Where is it? i can't found it. thanks!

**Comment by V1kt0r87** [comment]
@sephra-dev Where is it? i can't found it. thanks!

**Comment by kikimora52** [comment]
I am having this issue when creating new account sync'd from FreeIPA. I found the below error coming from the docker logs

`Fatal error: Uncaught InvalidArgumentException: Invalid CRON field value */0 at position 0 in /var/www/html/includes/libraries/Cron/CronExpression.php on line 231`

I have not troubleshot further and I made a comment to the called line reran scheduler.php and it finished creating the account.

This is not a fix, but a work around. I needed something working quickly and I really like this product so far.
